src/main/java/edu/kis/powp/jobs2d/features/FeatureManager.java

```diff
@@ -0,0 +1,36 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.powp.appbase.Application;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FeatureManager {
+	private static Map<Class, Feature> featureMap = new HashMap<>();
+
+	public static void addFeature(Feature feature){
+		featureMap.put(feature.getClass(), feature);
+	}
+
+	public static void addFeatures(Feature ... features){
+		for(Feature feature:features){
+			addFeature(feature);
+		}
+	}
+
+	public static Feature getFeature(Class type){
```

Comment:
Please try using templates to avoid casting the result:
public static <T extends Feature> T getFeature(Class<T> type)